### DISCUSSION TITLE
No possibility to redefine default/configured settings by admin/end users during export

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#82] [Q&A] [answered]
### Current Behavior

No possibility to redefine default/configured package settings by admin/end users in document Properties side bar or in export pdf live report page or live document export(dle button).Please check the below screenshots
![image](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/d1f9647a-0fbc-4835-918a-43d76065c7c4)
![image](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/411080f4-9858-4dc1-871f-ba4a82e5560c)

### Expected Behavior

below redefining settings are missing everywhere
![image](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/45649812/0da16310-314c-45f4-983a-d19bc8fd338d)

### Steps To Reproduce

please try to check the possibility to redefine the settings during export in document propertis or report pdf export button or dle button

### Environment - OS

Windows

### Polarion version

2404

### Extension Version

https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/releases/tag/v5.5.0

**Comment by yurtsevich-sbb** [comment, marked as answer]
This is not an issue. PDF-exporter settings (Polarion Administration portal) have an option to expose/hide settings to be redefined on UI. Please change this option yourself or ask your administrator:

![Screenshot 2024-07-10 at 10 21 33](https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/140163745/ace348c1-4108-4bae-95d9-6f3332b7662d)

**Comment by ukrishne** [comment]
thanks for the clarrification
